test/EVMHost.cpp

```diff
@@ -137,6 +137,14 @@ EVMHost::EVMHost(langutil::EVMVersion _evmVersion, evmc::VM& _vm):
 	// The minimum value of basefee
 	tx_context.block_base_fee = evmc::bytes32{7};
 
+	static evmc_bytes32 const blob_hashes_array[] = {
+		0x0100000000000000000000000000000000000000000000000000000000000001_bytes32,
+		0x0100000000000000000000000000000000000000000000000000000000000002_bytes32
+	};
+
+	tx_context.blob_hashes = blob_hashes_array;
+	tx_context.blob_hashes_count = sizeof(blob_hashes_array) / sizeof(blob_hashes_array[0]);
```

Comment:
Can we simplify it even more?
```suggestion
	tx_context.blob_hashes = {
		0x0100000000000000000000000000000000000000000000000000000000000001_bytes32,
		0x0100000000000000000000000000000000000000000000000000000000000002_bytes32
	};
	tx_context.blob_hashes_count = 2;
```

Reply:
Oh, actually not, because `tx_context.blob_hashes` is a pointer to `evmc_bytes32`. But the count can be hardcoded for sure.

Reply:
Well, if we have to have the separate variable, then having the count set automatically based on it was actually better :) Now that I think of it, I probably could have used `tx_context.blob_hashes()` in my suggestion too.

Reply:
Actually, looking at it again, I see why you wanted to put it in the class initially. I was assuming `tx_context.blob_hashes` was holding a copy and thought you made the variable `static` only as a micro-optimization.

If it's a pointer and we have to hold on to the memory then yeah, the original would be fine. But I guess hiding it as a static variable in a function is fine too.